Assets/HoloToolkit/SpatialUnderstanding/Readme.md

```diff
@@ -0,0 +1,193 @@
+Copied from [Spatial Mapping In Unity](https://developer.microsoft.com/en-us/windows/holographic/spatial_mapping_in_unity) page of the Microsoft Dev Center Holographic Documentation.
```

Comment:
Do you think it's better to just point the Readme to the documentation page?
This way if the main doc page gets updated, this page will not fall behind.

Also perhaps link the case study from this page?